### ISSUE TITLE
Support EU based rev.ai accounts

### ISSUE THREAD

**Issue by ronnyroeller** [revdotcom/revai-node-sdk#152]
### Issue

EU based accounts require a different base URL ([source](https://docs.rev.ai/api/global-deployments/)):
![image](https://user-images.githubusercontent.com/9339055/200775268-75c15ce1-ec94-4cf8-a9a5-a4e74490cb3a.png)

Yet, the SDK hardcodes the US based URL: https://github.com/revdotcom/revai-node-sdk/blob/a250658acd25e86a463bef419bd99d813f280b34/src/api-client.ts#L31

### Workaround

It's possible to overwrite the `apiHandler` field of the client (by plugging into the internal API):

```js
import { RevAiApiClient } from 'revai-node-sdk';
import { ApiRequestHandler } from 'revai-node-sdk/src/api-request-handler';

...

const client = new RevAiApiClient(accessToken);
client.apiHandler = new ApiRequestHandler('https://ec1.api.rev.ai/speechtotext/v1/', accessToken);
```

**Comment by amikofalvy** [comment]
This should be fixed with today's release of v3.6.0.

Your proposed solution was considered but we opted for a different solution. Please refer to the example found here for how to use the node sdk with the EU region: https://github.com/revdotcom/revai-node-sdk#usage

**Comment by ronnyroeller** [comment]
Thanks a lot for the quick fix, @amikofalvy ! Works like a charm.
